packages/framework-types/src/concepts/register.ts

```diff
@@ -1,6 +1,9 @@
 import { EventInterface } from './event'
 import { UserEnvelope, ContextEnvelope } from '../envelope'
 import { UUID } from './uuid'
+import { BoosterConfig } from '../config'
+
+export type FlusherFunction = (config: BoosterConfig, eventList: Array<EventInterface>, requestID: UUID, currentUser?: UserEnvelope) => Promise<void>
```

Comment:
We can make this function to have the same signature of `RegisterHandler.handle` and avoid the need of the `RegisterHandler.flush` method 😃